### PR TITLE
aligned checkboxes on teams page vertically

### LINKED ISSUE #233: Center Checkboxes
image

- Checkboxes should be vertically centered with the row

### PR DESCRIPTION
**Pull request by isabellecelo**
Screenshot 2023-08-16 at 10 58 00 AM

closes Center Checkboxes

### CODE REVIEW

File: src/components/Admin/Team.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const Teams = ({
     <div className="w-full font-poppins">
       <Col
         className={
-          "border-t-[1px] border-hackathon-gray flex w-full focus:!ring-0 focus:!bg-hackathon-green-100 " +
+          "border-t-[1px] border-hackathon-gray flex w-full focus:!ring-0 focus:!bg-hackathon-green-100 align-items-center " +
```
Comment (shahdivyank) [inline src/components/Admin/Team.jsx:46]: curious, what does the css look like for this ?

**Review by shahdivyank — CHANGES_REQUESTED** [review]

File: src/components/Admin/Team.jsx (hunk 1 above)
Comment (shahdivyank) [reply src/components/Admin/Team.jsx:46]: Can you put the CSS translation as a comment and then I can approve it

**Comment by shahdivyank** [comment]
Fix merge conflict

**Review by menthy-wu — DISMISSED** [review]

**Comment by shahdivyank** [comment]
Only the checkbox should be aligned to the center, links/email and everything else should be aligned to the top

**Comment by shahdivyank** [comment]
This is not needed anymore

**Pull request closed without merging**